### PR TITLE
Added dashboard as wireframe

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Created wireframe like dashboard.

**Sample :**

image

### CODE REVIEW

**Comment by Harikarthyk** [comment]
Please attach the preview or sample outcome.

File: application/backend/src/main/resources/apps/typerefinery/web_root/main.css [hunk 1]
```diff
@@ -155,4 +156,9 @@
 
 .d3Graph {
   width: 100%;
+}
+/* //new dashborad css*/
+.section.hl-page-section.hl-grid.hl-cols-12 {
+  align-items: flex-start;
+  margin-top: -67px;
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/web_root/main.css:163]: Did you check whether this does not affect any other components?

Comment (Rahul139-stack) [reply application/backend/src/main/resources/apps/typerefinery/web_root/main.css:163]: Done

**Comment by Harikarthyk** [comment]
Update outcome.

image

**Review by Harikarthyk — APPROVED** [review]

**Pull request merged**